File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/internalapi/android/os/ServiceManagerIATest.java
package me.ycdev.android.lib.common.internalapi.android.os;

import org.junit.Test;
import org.junit.runner.RunWith;

import androidx.test.ext.junit.runners.AndroidJUnit4;

import static org.junit.Assert.assertTrue;

@RunWith(AndroidJUnit4.class)
public class ServiceManagerIATest {
    @Test
    public void test_getService() {
        assertTrue(ServiceManagerIA.checkReflect_getService());
    }

    @Test
    public void test_checkService() {
        assertTrue(ServiceManagerIA.checkReflect_checkService());
    }

    @Test
    public void test_addService() {
        assertTrue(ServiceManagerIA.checkReflect_addService());
    }

    @Test
    public void test_listServices() {
        assertTrue(ServiceManagerIA.checkReflect_listServices());
    }
}

File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/internalapi/android/os/ServiceManagerIATest.kt
package me.ycdev.android.lib.common.internalapi.android.os

import org.junit.Test
import org.junit.runner.RunWith

import androidx.test.ext.junit.runners.AndroidJUnit4

import org.junit.Assert.assertTrue

@RunWith(AndroidJUnit4::class)
class ServiceManagerIATest {
    @Test
    fun test_getService() {
        assertTrue(ServiceManagerIA.checkReflectGetService())
    }

    @Test
    fun test_checkService() {
        assertTrue(ServiceManagerIA.checkReflectCheckService())
    }

    @Test
    fun test_addService() {
        assertTrue(ServiceManagerIA.checkReflectAddService())
    }

    @Test
    fun test_listServices() {
        assertTrue(ServiceManagerIA.checkReflectListServices())
    }
}
